obspy/io/seiscomp/inventory.py

```diff
@@ -1000,7 +1000,7 @@ def _read_float_var(elem, cls, unit=False, datum=False, additional_mapping={}):
     normally ObsPy would read this directly from a tag, but with different
     tag names this is no longer possible; instead we just pass the value
     and not the tag name. We always set the unit/datum/uncertainties to None
-    because they are not provided by scml ?
+    because they are not provided by SCML ?
```

Comment:
Question mark is confusing. Is this actually a question? Is so, it should be answered before providing code. See also above.

"provided by SCML" seems misleading. Either "provided by files in SCML" While, the statement is correct for unit, times in SCML do know value and uncertainty.

Reply:
that was a legacy comment... agree about the ?s, have removed. 

not knowing how accurate this still is, I would change to: `We set the unit/datum/uncertainties to None because they are not present in SCML XSD.`